stubs/argparse.pyi

```diff
@@ -1,3 +1,6 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: Ansible Project, 2020
+
```

Comment:
Looking at e411747, it seems this was copied from https://github.com/python/typeshed, which is licensed under `Apache-2.0`.

Reply:
Fixed in fda7f4711659c88b5309a76176922d572e64fde6.

Reply:
Thanks for spotting this one! (It also appears in antsibull-core.)